pyproject.toml

```diff
@@ -17,9 +17,9 @@ python = ">=3.8,<4.0"
 abnf = "1.1.1"
 web3 = "^6.0.0"
 python-dateutil = "2.8.2"
-eth-account = "^0.8.0"
-eth-utils = "^2.2.0"
-eth-typing = "^3.4.0"
+eth-account = ">=0.8.0,<0.11"
+eth-utils = ">=2.2.0,<2.4"
+eth-typing = ">=3.4.0,<3.6"
```

Comment:
The caret is for semver, these changes effectively restrict the range more

Reply:
Thanks, I've been operating under a bad assumption about the behavior of carets.  I'll revert.

However, it still doesn't explain what brought me here.  :thinking: